### DISCUSSION TITLE
Feature Idea - Allow users to select the image resolution

### DISCUSSION THREAD

**Discussion by Karthik-315** [subhoghoshX/laureate#42] [Ideas]
As per the current design, when a user copies or saves an image, it is always set with a scale (quality) of 3.

I am thinking of an additional functionality where the user can select the scale level they want, like having a drop-down each, near the two buttons (Save and Copy), where they can select the quality they want (maybe scale values from 1x to 5x). 

Let me know if you like it and your thoughts on this @subhoghoshX.

**Comment by subhoghoshX** [comment]
Not quite sure if users really want that kind of feature to be honest.

If I add it, I'll add the option in the side panel as in Figma. Adding it to the button might make the design look bad.

Not sure. Let's wait and see if anyone else has anything to add.

**Reply by Karthik-315** [reply]
Cool dude. Makes sense to add it to the side panel instead of the buttons, if we are adding it at all. Let's see.
